pkgs/applications/networking/sniffers/wireshark/default.nix

```diff
@@ -98,7 +87,12 @@ in stdenv.mkDerivation {
     cp ../epan/dfilter/*.h $dev/include/epan/dfilter/
     cp ../wsutil/*.h $dev/include/wsutil/
     cp ../wiretap/*.h $dev/include/wiretap
-  '';
+
+  '' + ''
+      # to remove "cycle detected in the references"
```

Comment:
This is part of the else clause, so it didn't run on darwin.

Reply:
arg my bad :/

Reply:
the precedence of if vs + is as little unexpected.